clients/web/src/utilities/EventStream.js

```diff
@@ -64,6 +73,7 @@ prototype.open = function () {
                 stream.trigger('g:error', e);
                 return;
             }
+            window.localStorage.setItem('sseTimestamp', obj._girderTime);
```

Comment:
My understanding is that localStorage is less likely to be allowed than cookies, would it be safer to store as a cookie? (If it isn't much more effort for us, that is)